### PR TITLE
Show standard deviation of inflammation data

### PR DESCRIPTION
**Pull request by NiekBrouwer98**
We implemented this feature in an earlier session and it fulfills requirement SR1.1.1

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
```
Comment (fleur-petit) [inline tests/test_models.py:42]: ziet er goed uit!

File: inflammation/models.py [hunk 2]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (fleur-petit) [inline inflammation/models.py]: perhaps the name of the function can be clearer?

Comment (fleur-petit) [inline inflammation/models.py]:
```suggestion
def standard_deviation(data)
```

File: inflammation/models.py [hunk 3]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
```
Comment (fleur-petit) [inline inflammation/models.py]: waarom heet deze s_dev2?

**Review by fleur-petit — COMMENTED** [review]
Minor suggesties, ziet er over all goed uit.

File: inflammation/models.py (hunk 2 above)
Comment (NiekBrouwer98) [reply inflammation/models.py]: Resolved this in commit ca302b1

Comment (NiekBrouwer98) [reply inflammation/models.py]: Resolved this also in commit ca302b1

File: inflammation/models.py (hunk 3 above)
Comment (NiekBrouwer98) [reply inflammation/models.py]: also resolved in commit ca302b1

**Review by fleur-petit — APPROVED** [review]

**Pull request merged**